src/main/java/duke/command/AddEventCommand.java

```diff
@@ -0,0 +1,57 @@
+package duke.command;
+
+import duke.*;
+import duke.task.Event;
+import duke.task.TaskList;
+
+/**
+ * Adds an event to the task list.
+ */
+public class AddEventCommand extends Command {
+    private String description;
+    private DateHelper datetime;
+
+    private static final String DESC_RESPONSE = "Oops! \\(@.@)/ You have not keyed in a description for the task!\n" +
+            "Let's try again ~(^.^)~\n" +
+            "Type 'help' if you need to know how to use this command";
```

Comment:
With regards to the coding standard, the break should be before an operator.  I noticed the same issue in several other places too. 
Perhaps such as :
 <code> "Oops! \\(@.@)/ You have not keyed in a description for the task!\n" </code>
         <code>     + "Let's try again (^.^)\n" </code>
         <code>   +  "Type 'help' if you need to know how to use this command"; </code> ? 🤔